service/src/main/java/bio/terra/cbas/common/MetricsUtil.java

```diff
@@ -26,6 +26,8 @@ private MetricsUtil() {}
   public static final String METRICS_PREFIX = "terra/cbas/";
 
   public static final String OUTBOUND_REQUEST_METRICS = METRICS_PREFIX + "request/outbound";
+  public static final String RUNS_SUBMITTED_SUCCESSFULLY_PER_RUN_SET_METRICS =
```

Comment:
I had meant to put this question for reviewers and forgot about it. I think it would be great to record both success and failure Run submissions. How do I use the tag feature in metrics?